packages/server/lib/controllers/v1/flow/upgrade/preBuilt.ts

```diff
@@ -0,0 +1,49 @@
+import { z } from 'zod';
+import { asyncWrapper } from '../../../../utils/asyncWrapper.js';
+import type { UpgradePreBuiltFlow, IncomingFlowConfigUpgrade } from '@nangohq/types';
+import { requireEmptyQuery, zodErrorToHTTP } from '@nangohq/utils';
+import { flowConfig } from '../../../sync/deploy/postConfirmation.js';
+import { logContextGetter } from '@nangohq/logs';
+import { upgradePreBuilt as upgradePrebuiltFlow } from '@nangohq/shared';
+
+const validation = flowConfig.extend({
+    id: z.string(),
+    upgrade_version: z.string(),
+    last_deployed: z.string(),
+    is_public: z.boolean(),
+    pre_built: z.boolean()
+});
+
+export const upgradePreBuilt = asyncWrapper<UpgradePreBuiltFlow>(async (req, res) => {
```

Comment:
we should agree on a definitive naming convention and filename for endpoints.
I really like  when the filepath matches the endpoint path with the file name including the method.  For instance the endpoint here could be `PUT /api/v1/flow/prebuilt` and the file `.../controllers/v1/flow/putPreBuild.ts`.
I like it because when looking for the endpoint code you can directly look for the filepath without having to first look into `routes` and all the code relative to a subpath is neatly organized in the same subdir

Reply:
I hear ya, updated